techlibs/xilinx/lutrams.txt

```diff
@@ -1,4 +1,17 @@
 
+bram $__XILINX_RAM16X1D
```

Comment:
Why this rule?  `RAM16X1D` is a LUT4 architecture leftover, and is not natively supported by LUT6 architectures — it will just be converted to `RAM32X1D`.